### ISSUE TITLE
projects mapped to workspace root

### ISSUE THREAD

**Issue by rogermartensson** [rozpuszczalny/vscode-redmine#16]
Nice extension. I hope you will have time to add functionality and UI-extension. 

I'm using multi-root workspace against several internal repositorys. These repositorys each have their own project in our redmine server.

For example. In my multi-root workspace I have cloned git-repositories that for some actions would like to map to specific projects in redmine. One such action is New Issue.

**Comment by rozpuszczalny** [comment]
I have to check how to implement these things. I'll be doing this with issue #15 in mind (eg. issues assigned to me in defined project), to make project definition an efficient tool.

**Comment by rozpuszczalny** [comment]
I've looked up how multi-root workspaces work under the hood and it's required to make a rework. So this issue must be done before anything moves forward.
